js/src/common/components/ModalManager.js

```diff
@@ -53,7 +53,12 @@ export default class ModalManager extends Component {
 
     m.redraw(true);
 
-    this.$().modal({backdrop: this.component.isDismissible() ? true : 'static'}).modal('show');
+    var dismissible = !!this.component.isDismissible();
```

Comment:
Using const or let changes the scope of the declared variable over var.

```suggestion
    const dismissible = !!this.component.isDismissible();
```

Reply:
Not sure why this suggestion was resolved and not applied...?